CHANGELOG.rst

```diff
@@ -1,6 +1,11 @@
 pyFLAC Changelog
 ----------------
 
+**v2.0.0**
```

Comment:
I don't know if we've decided on a versioning scheme, but to me, this doesn't feel like enough to bump the major version

Reply:
The interface has changed by renaming `callback` to `write_callback`, so if we're following semver we need to go to v2.